src/components/FlowRunGraph.vue

```diff
@@ -105,8 +111,14 @@
 
   const config = computed<RunGraphConfig>(() => ({
     runId: props.flowRun.id,
-    fetch: (id) => {
-      return api.flowRuns.getFlowRunsGraph(id, { nestedTaskRunGraphs: true })
+    fetch: async (id) => {
+      try {
+        return await api.flowRuns.getFlowRunsGraph(id, { nestedTaskRunGraphs: true })
+      } catch (error) {
+        failed.value = true
+        errorMessage.value = getApiErrorMessage(error, 'An error occurred retrieving graph data')
+        throw error
```

Comment:
```suggestion
```
I don't think we need to re throw this error!

Reply:
I'm going to leave in because otherwise I think I need to update the type of RunGraphConfig.fetch to accept undefined and I'm not certain that's a useful/wanted change.